Sources/Turf/GeoJSON.swift

```diff
@@ -3,56 +3,96 @@ import Foundation
 import CoreLocation
 #endif
 
+public typealias FeatureIdentifierNumber = Numeric & Codable
+public typealias FeatureIdentifierString = StringProtocol & Codable
 
-public class BaseFeature: Codable {
-    private enum CodingKeys: String, CodingKey {
-        case properties
+public struct FeatureIdentifier<T: FeatureIdentifierNumber, U: FeatureIdentifierString>: Codable {
```

Comment:
Would it be possible to represent the identifier as an enumeration with associated values (with cases `number(Numeric)` and `string(StringProtocol)`)? I think that would be easier for a developer to understand than a struct with template parameters. Or would that be incompatible with Codable?